### PR TITLE
fix(deps): Readd fixed style import from nextcloud/dialogs

### LINKED ISSUE #10854: Fix nextcloud/dialogs usage
> As per https://github.com/nextcloud/files_retention/pull/399#discussion_r1388088157 I misunderstood it and the file name was just wrong.

_Originally posted by @nickvergessen in https://github.com/nextcloud/spreed/pull/10821#discussion_r1388184729_

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* Fix #10854 

## 🖌️ UI Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖌️ Design was reviewed, approved or inspired by the design team
- [x] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -67,6 +67,9 @@ import { EventBus } from './services/EventBus.js'
 import { leaveConversationSync } from './services/participantsService.js'
 import { signalingKill } from './utils/webrtc/index.js'
 
+// Styles
+import '@nextcloud/dialogs/style.css'
+
```
[the remark this answers is not in the document]
Comment (ShGKme) [reply src/App.vue:72]: In general, the most correct way is to import styles in the same files where `@nextcloud/dialogs` is imported.

Aka
```js
import { showError } from '@nextcloud/dialogs'
import '@nextcloud/dialogs/style.css'
```

Then we don't need to think about in what entry point we need and do not need the styles. (For example, "is it needed in recording? In collections or flow?").

[the remark this answers is not in the document]
Comment (ShGKme) [reply src/App.vue:72]: If we are sure we know where the styles are needed (or don't care about redundant styles), we can also use `init.js` as a common file for most of the entry points. Then we would need the import only in 2 places:
- `init.js`
- `mainRecording.js` (it doesn't use `init`).

[the remark this answers is not in the document]
Comment (nickvergessen) [reply src/App.vue:72]: Feel free to take over, I was just reverting my "breakage" from #10821

[the remark this answers is not in the document]
Comment (ShGKme) [reply src/App.vue:72]: Discussing with @susnux possible solutions.

Currently I see 3 solutions:
1. Always import `@nextcloud/dialogs` and `@nextcloud/dialogs/style.css` together
   - ```js
     import { showError } from '@nextcloud/dialogs'
     import '@nextcloud/dialogs/style.css'
     ```
   - Styles would be imported in 62 files
   - No guarantee that new `@nextcloud/dialogs` will import styles
2. Import styles in every entry point that indirectly uses `@nextcloud/dialogs`
   - No guarantee that no entry point is missed and no is redundant
3. Add a local module with styles and re-export + forbid direct import of `@nextcloud/dialogs` with ESLint
   - ```js
     // utils/nextcloudDialogs.js
     import '@nextcloud/dialogs/style.css'
     export * from '@nextcloud/dialogs'
     ```
   - Guarantees that styles are always and only imported when needed
   - Looks a bit dirty that dialogs are not imported directly
4. Import in all entry points or in global styles like `icons.css` (suggestion from @susnux, used in server)
   - Adds redundant styles to places without dialogs

**Comment by ShGKme** [comment]
PR updated.

`@nextcloud/dialogs` requires importing styles and supposed to be used as
```js
import { showError } from '@nextcloud/dialogs'
import '@nextcloud/dialogs/style.css'
```

But we use dialogs in **60+ modules**. It is easy to forget import styles each time.

Importing styles once per entry point is enough to use dialogs on the entry point.

In this commit, styles are imported only in entry points were they are needed **by manual check**.

- Imported via common `init.js`
  - main: `main.js`
  - files-sidebar: `mainFilesSidebar.js + mainFilesSidebarLoader.js`
  - public-share-auth-sidebar: `mainPublicShareAuthSidebar.js`
  - public-share-sidebar: `mainPublicShareSidebar.js`
- Imported manually
  - maps: `maps.js`
  - deck: `deck.js`
  - admin-settings: `mainAdminSettings.js`
  - recording: `mainRecording.js`
- Not imported, don't `@nextcloud/dialogs`:
  - dashboard: `dashboard.js`
  - flow: `flow.js`
  - collections: `collections.js`

This is not very reliable, because we may add a new entry point, or start to use dialogs in some existing entry point without dialogs, or stop using dialogs in some entry point.

But it's simple and clear.

Alternative solutions:
- Import everywhere with webpack config no matter if `@nextcloud/dialogs` are used
- Use `@nextcloud/dialogs` via wrapper module and forbid direct import by ESLint rule:
  ```js
  // utils/nextcloudDialogs.js
  import '@nextcloud/dialogs/style.css'
  export * from '@nextcloud/dialogs'
  ```

**Review by Antreesy — APPROVED** [review]
Styles for toast messages and file pickers are loaded and referred to spreed repo.
There is another style source from server with 404, but it seems to not have an effect on Talk app 

Tested for:
- main.js
- mainAdminSettings.js
- init.js (file sidebar)

**Pull request merged**